cpu/sam0_common/periph/uart.c

```diff
@@ -129,6 +129,24 @@ static void _set_baud(uart_t uart, uint32_t baudrate, uint32_t f_src)
 #endif
 }
 
+void uart_enable_tx(uart_t uart)
+{
+    /* configure RX pin */
+    if (uart_config[uart].tx_pin != GPIO_UNDEF) {
+        gpio_set(uart_config[uart].tx_pin);
+        gpio_init_mux(uart_config[uart].tx_pin, uart_config[uart].mux);
+    }
+}
+
+void uart_disable_tx(uart_t uart)
+{
+    /* configure RX pin */
+    if (uart_config[uart].tx_pin != GPIO_UNDEF) {
+        gpio_clear(uart_config[uart].tx_pin);
```

Comment:
Shouldn't this only be a clear when the UART has inverted signal levels, otherwise be a set?

Reply:
Ah turns out I don't even need this when the GPIO is not configured as output.